File: ClientApp/src/app/clases/usuario.ts
export class usuario{
    id : number;
    nombre : string;
    codigo : string;
    codigoayuda: string;
    correo : string;
    estado:number;
    
    constructor(obj:any){
        this.id = +obj.id;
        this.nombre = obj.nombre;
        this.codigo = obj.codigo;
        this.codigoayuda= obj.codigoayuda;
        this.correo = obj.correo;
        this.estado=+ obj.estado;
    }

}

File: modelo/usuario.cs
using System.Data;
using System;
public class usuario : oObjeto
{
    public string Nombre { get; set; }
    public string Codigo { get; set; }
    public string Codigoayuda{ get; set; }
    public string Correo { get; set; }
    public int Estado{get;set;}
    public usuario() { }
    public usuario(DataRow dr)
    {
        this.Id = Convert.ToInt32(dr["id"]);
        this.Nombre = dr["nombre"].ToString();
        this.Codigo = dr["codigo"].ToString();
        this.Codigoayuda = dr["codigoayuda"].ToString();
        this.Correo = dr["correo"].ToString();
        this.Estado=Convert.ToInt32(dr["estado"]);
    }

}